src/lib/network/wicked.rb

```diff
@@ -83,7 +84,16 @@ def query_wicked(iface, query)
     #
     # @return [Array <String>] array of interface names
     def ibft_interfaces
-      Yast::Execute.stdout.locally!(IBFT_CMD, "-l").gsub("\n", " ").split(" ").uniq
+      Yast::Execute.stdout.locally!(IBFT_CMD, "-l").gsub("\n", " ").split(/\s+/).uniq
+    end
+
+    # Returns an array of interface names which are configured via firmware
+    #
+    # @return [Array <String>] array of interface names
+    def firmware_interfaces
+      output = Yast::Execute.stdout.locally!(WICKED_PATH, "firmware", "interfaces")
+      interfaces = output.gsub(/^(ibft|nbft|redfish)\s+/, "").gsub("\n", " ").split(/\s+/)
```

Comment:
I don't think `gsub("\n", " ")` is needed since `\s` includes `\n`.

Reply:
In fact it is not :+1:

Reply:
Did you see my previous command about parsing based on the format (columns) instead of strings (technology names). https://github.com/yast/yast-network/pull/1319/files#r1086825042

Maybe it deserves a comment in the code (FIXME, TODO, whatever).

Reply:
So do you prefer something like

`
 interfaces = output.gsub(/^\w+\s+/, "").split(/\s+/)
`

Reply:
BTW we probably would like to modify it in the future as it would be nice to now which interfaces are configured by which technology to show it in the overview summary.